tests/utils.py

```diff
@@ -248,6 +249,19 @@
     ['1', 0.7],
     ['2', 0.2],
     ['3', 0.1],
-], columns=['item', 'percentage'])
+], columns=['user', 'percentage'])
+
+item_feature = pd.DataFrame([
+    ['1', 'pop'],
+    ['2', 'pop'],
+    ['3', 'electronic'],
+    ['4', 'rock'],
+], columns=['item', 'genre'])
 
+user_feature = pd.DataFrame([
+    ['1', 'm'],
+    ['2', 'f'],
+    ['3', None],
+    ['4', np.nan],
```

Comment:
- `np.nan` (literally `NotANumber`) is used for number with value equal to `null`
- `None` is used for any type of value (`string`, `object`, ...) equal to `null`
Choose the one you think is fine in this case, to be more realistic with our tests

Reply:
I explicitly used both to check that they were both correctly converted to -1 in the tests.